spec/stub_spec.lua

```diff
@@ -98,4 +98,13 @@ describe("Tests dealing with stubs", function()
     assert.is.equal("foo", test.key())
   end)
 
+  it("returns multiple given values", function()
+    stub(test, "key", "foo", "bar")
+
+    arg1, arg2 = test.key()
+
+    assert.is.equal("foo", arg1)
+    assert.is.equal("bar", arg2)
+  end)
+
```

Comment:
Ah ok. Great, thanks for the heads up. Updated the spec to include a nil in the middle of the return.